src/elements.ts

```diff
@@ -1354,51 +1354,105 @@ const SVG_TAGS = ['path', 'rect', 'circle', 'ellipse', 'polygon', 'polyline',
   'g', 'defs', 'marker', 'line', 'text', 'tspan', 'pattern', 'mask', 'svg',
   'foreignObject', 'image', 'use'];
 
+type SvgTag =
+  'path'|
+  'rect'|
+  'circle'|
+  'ellipse'|
+  'polygon'|
+  'polyline'|
+  'g'|
+  'defs'|
+  'marker'|
+  'line'|
+  'text'|
+  'tspan'|
+  'pattern'|
+  'mask'|
+  'svg'|
+  'foreignObject'|
+  'image'|
+  'use';
```

Comment:
Rather than repeating the strings here, lets use this:

``` ts
const SVG_TAGS = ['path', 'rect', 'circle'] as const;
type SvgTag = typeof SVG_TAGS[number];
```

Reply:
Good idea :)